utils/ofi_debug_ip.patch

```diff
@@ -0,0 +1,140 @@
+diff --git a/prov/tcp/src/xnet.h b/prov/tcp/src/xnet.h
+index ee16d7e3a..ae3567e1f 100644
+--- a/prov/tcp/src/xnet.h
++++ b/prov/tcp/src/xnet.h
+@@ -754,4 +754,26 @@ int xnet_rdm_ops_open(struct fid *fid, const char *name,
+ 	FI_WARN(&xnet_prov, subsystem, log_str "%s (%d)\n", \
+ 		fi_strerror((int) -(err)), (int) err)
+ 
++#define XNET_OUTPUT_ERR_WARN(ep, err)		\
++do {						\
++	struct sockaddr *addr;			\
```

Comment:
In macros, it is better to use more unique names like __addr in case user has addr.  Otherwise, compiler will warn about same named variable in scope.  We hit this with rc in other macros

Reply:
ah, ok